### ISSUE TITLE
Tool reports that ItemsPath is not a valid property for Map activities

### ISSUE THREAD

**Issue by theshadow** [awslabs/statelint#37]
While executing the tool over a JSON blob I found that the tool is reporting an error when defining `ItemsPath`. This appears to be a valid property and probably shouldn't raise an error. 

```
2 errors:
 Field "ItemsPath" not allowed in State Machine.States.MapTenantsToDevices
 Field "ItemsPath" not allowed in State Machine.States.MapTenantsToDevices.Iterator.States.MapDeviceToQueryAndPublish
```

**Comment by theshadow** [comment]
According to https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-map-state.html `ItemsPath` is valid.

**Comment by wong-a** [comment]
This was fixed and merged in #30 . We'll be releasing this to ruby gems shortly in v0.4.1. In the meantime, you can build the gem in from the source to use locally.

**Comment by wong-a** [comment]
Version 0.4.1 is now on ruby gems https://rubygems.org/gems/statelint/versions/0.4.1
